### ISSUE TITLE
Double tap "control" key vs "X" button on TouchBar

### ISSUE THREAD

**Issue by Looki75** [pock/pock#185]
**Context:**
No matter about actions which lead to situation 1:
1. Start with "pock strip" activated and showing on TouchBar
2. Close the "pock strip" with the "X" button on TouchBar > the "pock strip" closes 👍
3. Double tap the "control" keyboard physical key

**Expected behavior:**
>> Double tapping "control" key should open the "pock strip" back

**Real behavior:**
>> Double tapping "control" key has no effect
>> Double tapping "control" a second time opens the "pock strip"

**Comment by nwesthoff** [comment]
Close and reopen Pock, same issue as https://github.com/pigigaldi/Pock/issues/96, probably.

**Comment by Looki75** [comment]
> Close and reopen Pock, same issue as #96, probably.

Well I don't think the same issue.

Maybe both issues are related one to another, but the issue I raised seems different because:
1. Closing/re-opening Pock doesn't change anything. The bug is still here
2. I have the bug I reported on a constant basis, i.e. at fresh start, after restarting Pock, after rebooting MacBook, etc...
3. I never experienced the "Pock ribbon not expanding on touching the Pock Icon in the strip" bug...

Maybe I was not clear enough when explaining the bug...
Feel free to ask me questions as necessary.

Thanks
